OWNERS

```diff
@@ -0,0 +1,28 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+admins:
+
+- mattklein123
+- pydctw
+- skasisom6
+- sunjayBhatia
+- Xunzhuo
+- zinuga
+
+maintainers:
+
+- AliceProxy
```

Comment:
this list needs to be updated

Reply:
we need to rm `alexgervais` and `danehans`

Reply:
Done